pyof/foundation/base.py

```diff
@@ -366,15 +367,16 @@ def replace_pyof_version(module_fullname, version):
 
         Args:
             module_fullname (str): The fullname of the module
-                (e.g.: pyof.v0x01.common.header)
+                                   (e.g.: pyof.v0x01.common.header)
             version (str): The version to be 'inserted' on the module fullname.
 
         Returns:
-            None (None): if the requested version is the same as the one of the
-                module_fullname or if the module_fullname is not a 'OF version'
-                specific module.
-            new_module_fullname (str): The new module fullname, with the
-                replaced version, on the format "pyof.v0x01.common.header".
+            str  : module fullname
+                   The new module fullname, with the replaced version,
+                   on the format "pyof.v0x01.common.header". If the requested
+                   version is the same as the one of the module_fullname or if
+                   the module_fullname is not a 'OF version' specific module
+                   the return is None.
```

Comment:
_...module, returns None._ Is it common practice to make a short first line and write more comments in a new line?